hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupBasedLoadBalancer.java

```diff
@@ -76,7 +76,7 @@
   private MasterServices masterServices;
   private FavoredNodesManager favoredNodesManager;
   private volatile RSGroupInfoManager rsGroupInfoManager;
-  private LoadBalancer internalBalancer;
+  private volatile LoadBalancer internalBalancer;
```

Comment:
This belongs in this patch?

Reply:
In general, no.

But this is also a fix on the balancer implementation and this patch will be applied to all active branches I believe, so I think it is OK to also include these minor changes.